migrations/migration.template

```diff
@@ -0,0 +1,9 @@
+import * as knex from 'knex'
+
+export function up (knex: Knex): Knex.SchemaBuilder {
+
+}
+
+export function down (knex: Knex): Knex.SchemaBuilder {
+
```

Comment:
same as above but with the usage of `dropTableifExist`

Reply:
@eoln @spikerheado1234 same as above